tests/test_callbacks.py

```diff
@@ -0,0 +1,22 @@
+from lightning_pose.callbacks import UnfreezeBackbone
+
+def test_unfreeze_backbone():
+    unfreeze_backbone = UnfreezeBackbone(2)
+
+    # Test unfreezing at epoch 2.
```

Comment:
comment that expected behavior is zero lr until epoch 2? then specify why 1e-4. which is the set LR?